README.md

```diff
@@ -1,29 +1,42 @@
 # cdist
 
-**cdist** is a usable configuration management system.
-
-It adheres to the [**KISS principle**](https://en.wikipedia.org/wiki/KISS_principle)
-and is being used in small up to enterprise grade environments.
-
-For more information have a look at [**homepage**](https://cdi.st)
-or at **``docs/src``** for manual in **reStructuredText** format.
-
-## Contributing
-
-Pull requests can be made in both [upstream](https://code.ungleich.ch/ungleich-public/cdist)
-(managed by [**ungleich**](https://ungleich.ch)) and [GitHub](https://github.com/ungleich/cdist).
-
-Issues can be made and other project management activites happen
-[**only in upstream**](https://code.ungleich.ch/ungleich-public/cdist)
-(needs [**ungleich** account](https://account.ungleich.ch)).
-
-For community-maintained types there is
-[**cdist-contrib** project](https://code.ungleich.ch/ungleich-public/cdist-contrib).
-
-## Participating
-
-IRC: ``#cdist`` @ [libera](https://libera.chat)
-
-Matrix: ``#cdist:ungleich.ch``
-
-Matrix and IRC are bridged.
+This is [community maintained](https://github.com/cdist-community)
+fork of Ungleich's [cdist](https://github.com/ungleich/cdist)
+(after [f061fb168ddacc894cb6e9882ff5c8ba002fadd8](https://github.com/ungleich/cdist/commit/f061fb168ddacc894cb6e9882ff5c8ba002fadd8)).
+
+Work is split between three repositories:
+
+* [cdist](https://github.com/cdist-community/cdist) - main implementation of **cdist core**.
+* [cdist-conf](https://github.com/cdist-community/cdist-conf) - **essential** explorers and types.
+* [cdist-extra](https://github.com/cdist-community/cdist-extra) - **non-essential** explorers, types, scripts, tools etc.
+
+What's the difference between essential and non-essential? Explorers and types,
+which are used to explore or change state (modify files and directories,
+install packages, manage services etc) of operating system and are not strictly
+related to some specific piece of software, are considered essential.
+
+## Getting Started
+
+Since this fork is still in early stages, there will be no versioned releases
+and everything is expected to be used straight from repository, because for now
+we are targeting power users of cdist, who know what they are doing.
+
+### Get core with essentials
+
+```bash
+git clone --recurse-submodules https://github.com/cdist-community/cdist
+ln -s "$(pwd)/cdist/bin/cdist" "$HOME/.local/bin/cdist"
+```
+
+### Get everything
+
+```bash
+git clone https://github.com/cdist-community/cdist
+git clone https://github.com/cdist-community/cdist-conf
+git clone https://github.com/cdist-community/cdist-extra
+ln -s "$(pwd)/cdist/bin/cdist" "$HOME/.local/bin/cdist"
+cat > "$HOME/.cdist.cfg" << EOF
```

Comment:
I think this is a bit dangerous. Someone who already has cdist installed and carelessly copy-pastes these lines into their terminal would overwrite their config.

I'd at least make it `>>"${HOME:?}/.cdist.cfg`.

Reply:
But we are targeting power users for now, right? :smile: I added warning in the beginning.